src/components/Hero/Hero.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Table from '../../assets/images/table.png';
+import {
+  HeroWrapper,
+  Title,
+  Subtitle,
+  HeroForm,
+  CTA,
+  Input,
+} from './Hero.style';
+
+function Hero({ title, subtitle, cta }) {
+  return (
+    <HeroWrapper>
+      <Title>{title}</Title>
+      <Subtitle>{subtitle}</Subtitle>
+      <HeroForm action="/search">
+        <CTA type="submit">{cta}</CTA>
+        <Input type="text" name="query" aria-label="Search" defaultValue="javascript" />
```

Comment:
There is actually no form in the hero section :-)

![image](https://user-images.githubusercontent.com/9722792/77847578-69bde900-71be-11ea-93dc-f014fd3bd985.png)

Reply:
Ah ok, now I understand. `r/javascript` is just a string. Clicking the button always leads to `/search?query=javascript`. This page is only for demonstration purposes so users can see immediately what the app is doing